Do not suppress errors for implicit function declarations
(which was done for https://rt.cpan.org/Ticket/Display.html?id=133493)
See: https://trac.macports.org/ticket/61630

--- lib/ExtUtils/MM_Darwin.pm.orig	2021-12-17 01:32:03.000000000 -0800
+++ lib/ExtUtils/MM_Darwin.pm	2021-12-17 12:00:38.000000000 -0800
@@ -62,7 +62,6 @@
     foreach (split /\n/, $base) {
         /^(\S*)\s*=\s*(\S*)$/ and $self->{$1} = $2;
     };
-    $self->{CCFLAGS} .= " -Wno-error=implicit-function-declaration";
 
     return $self->{CFLAGS} = qq{
 CCFLAGS = $self->{CCFLAGS}
